examples/assert-throws-error/src/main/ml-modules/root/lib/example.sjs

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const BOOM_QNAME = xs.QName("MATCH-BOOM");
+const BOOM_ERROR = "The values matched";
+
+function mightBoom(param1, param2) {
+  xdmp.log(`mightBoom; param1=${param1.toString()}; param2=${typeof param2}`);
+  if (param1 === param2) {
```

Comment:
> I'm wondering about the importance of handling this particular scenario

I got concerned about it because I think it's a pretty reasonable expectation that if I write a JavaScript function that works correctly and write JavaScript tests to pass in parameters, it ought to Do The Right Thing. In this case, it doesn't. Having an example that shows it working when it kinda doesn't could be misleading. I like the `param1.toString() === param2.toString()` change for the example, but with a comment that states why it's necessary and points to an issue on GitHub. And maybe the test can show the workaround of using something other than `assertThrowsError`. 

Thoughts?

Reply:
I'm thinking an SJS equivalent of test-helper.xqy is ultimately the right way to go. 

How about the use of toString() and a comment indicating e.g. "When an SJS test uses test-helper.xqy and passes a function, the parameters are wrapped as objects which may impact how the application code has to be written or has to be tested. A future SJS version of test-helper.xqy will avoid this issue. "

Reply:
I like it.

Reply:
Updated. Tests now pass